packages/grid/src/types/index.ts

```diff
@@ -158,3 +157,69 @@ export interface IRowProps extends HTMLAttributes<HTMLDivElement> {
   /** Applies the `flex-wrap` container property to the row on extra-large screens */
   wrapXl?: Wrap;
 }
+
+export const ORIENTATION = ['top', 'bottom', 'start', 'end'] as const;
+export type Orientation = typeof ORIENTATION[number];
+
+export interface IPaneProviderProps {
+  /** Provides the total width, in `px` units, of all panes in the layout */
+  totalPanesWidth: number;
+  /** Provides the total height, in `px` units, of all panes in the layout */
+  totalPanesHeight: number;
+  /** Defines default row values, in `fr` units, for an uncontrolled layout. The values are keyed by splitter. */
+  defaultRowValues?: Record<string, number>;
+  /** Defines default column values, in `fr` units, for an uncontrolled layout. The values are keyed by splitter. */
+  defaultColumnValues?: Record<string, number>;
+  /** Defines row values, in `fr` units, for a controlled layout. The values are keyed by splitter. */
+  rowValues?: Record<string, number>;
+  /** Defines column values, in `fr` units, for a controlled layout. The values are keyed by splitter. */
+  columnValues?: Record<string, number>;
+  /**
+   * Handles splitter position changes
+   *
+   * @param rowValues The updated row values
+   * @param columnValues The updated column values
+   */
+  onChange?: (rowValues: Record<string, number>, columnValues: Record<string, number>) => void;
+  /** Surfaces render prop functions for applying splitter state to the supporting layout */
+  children?: ({
+    getLayoutValue,
+    getGridTemplateRows,
+    getGridTemplateColumns
+  }: {
+    /**
+     * Gets layout value by key.
+     *
+     * @param {string} splitterKey The splitter key to get a layout value for
+     * @param {boolean} isRow Determines whether to return the row value,
+     *        otherwise returns the column value by default
+     * @param {boolean} isPixels Determines whether to return a `px` value,
+     *        otherwise returns a `fr` unit by default
+     */
+    getLayoutValue: (splitterKey: string, isRow: boolean, isPixels?: boolean) => number;
+    /** Gets grid template rows track. Accepts optional units.
+     *
+     * @param {boolean} isPixels Determines whether to return a `px` value,
+     *        otherwise returns a `fr` unit by default
+     *
+     */
+    getGridTemplateRows: (isPixels?: boolean) => string;
+    /** Gets grid template columns track. Accepts optional units.
+     *
+     * @param {boolean} isPixels Determines whether to return a `px` value,
+     *        otherwise returns a `fr` unit by default
+     */
+    getGridTemplateColumns: (isPixels?: boolean) => string;
+  }) => any;
```

Comment:
```suggestion
 /**
   * Surfaces render prop functions for applying splitter state to the supporting layout
   *
   * @param getLayoutValue Gets layout value by splitter key
   * @param getGridTemplateRows Gets grid template rows track
   * @param getGridTemplateColumns Gets grid template columns track
   */
  children?: ({
    getLayoutValue,
    getGridTemplateRows,
    getGridTemplateColumns
  }: {
    getLayoutValue: (splitterKey: string, isRow: boolean, isPixels?: boolean) => number;
    getGridTemplateRows: (isPixels?: boolean) => string;
    getGridTemplateColumns: (isPixels?: boolean) => string;
  }) => ReactNode;
```

The suggested change adds clarity to the generated docs while removing unused JSDoc that is best covered by TypeScript. The improved `ReactNode` return value is going to necessitate some `spec` test updates.

Reply:
**After**

<img width="709" alt="Screen Shot 2022-05-02 at 10 31 55 AM" src="https://user-images.githubusercontent.com/143773/166252080-c42b5733-79b7-42ab-9458-7bb44304228e.png">

Reply:
Fixed in https://github.com/zendeskgarden/react-components/pull/1339/commits/7df0ecb7cc11861806fb2838667fa5947e7b80b2